### ISSUE TITLE
Expliticly specify whether the context is evaluation of English language texts, or whether multilinguality is aimed at

### ISSUE THREAD

**Issue by ljleppan** [evanmiltenburg/Shared-task-on-NLG-Evaluation#3]
Right now, the initial description includes e.g. the note `studying the difference between L1 and L2-speakers of English.'

Is the intent of the shared task to investigate evaluation of English only, or other languages as well? It's conceivable, perhaps even likely, that evaluation metrics behave differently on different languages depending on e.g. morphological complexity, freedom of word order, availability of resources such as synsets, parsers or possibly even just plain old text corpora.

I believe it would be important to explicitly address the applicability of the proposed metrics outside of English and other similar languages with high resource availability.

**Comment by evanmiltenburg** [comment]
Good point! I would love to see a discussion of evaluation in different languages.
